ngx-fudis/projects/ngx-fudis/src/lib/components/footer/readme.mdx

```diff
@@ -0,0 +1,32 @@
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import { linkTo } from '@storybook/addon-links';
+import { FooterComponent } from './footer.component';
+import * as FooterComponentStories from './footer.stories.ts';
+import { Source } from '@storybook/blocks';
+
+<Meta title="Components/Footer" />
+
+# Footer
+
+Fudis footer is designed to be used with content projection in listing [Fudis Link's](/docs/components-link--link) in a bottom of the page. Through content projection it's easier for developers to change related links depending on the application.
```

Comment:
Fudis Footer is static component to be used as application's footer in the bottom of the page.

It is designed to use [Fudis Link](/docs/components-link--link) components as its content.